sparkprs/models.py

```diff
@@ -87,6 +87,7 @@ class Issue(ndb.Model):
         ("MLlib", "mllib", "mllib"),
         ("GraphX", "graphx|pregel", "graphx"),
         ("Streaming", "stream|flume|kafka|twitter|zeromq", "streaming"),
+        ("R", "SparkR", "r"),
```

Comment:
For the filename regex, we might want to only match files in the top-level `R` folder or in the Java half of the SparkR, so how about `(^r/)|src/main/r/|api/r/`, which I think should cover all R-related sources?